packages/adapter-vercel/index.js

```diff
@@ -31,12 +31,24 @@ export default function (options) {
 			utils.log.minor('Generating serverless function...');
 			utils.copy(join(files, 'entry.js'), '.svelte-kit/vercel/entry.js');
 
+			const assets = new Set([...utils.build_data.static, ...utils.build_data.client]);
+
+			utils.build_data.static.forEach((file) => {
+				if (file.endsWith('/index.html')) {
```

Comment:
probably pedantic but to remove the magic number -11 I would probably do either:

```js
const index_filename = '/index.html'
if (file.endsWith(index_filename)) {
	assets.add(file.slice(0, -(index_filename.length)));
}
```

or, even

```js
assets.add(path.dirname(file))
```

Reply:
i tend to think these sorts of magic numbers make sense in context, but maybe that's just me. an alternative is to just comment it

```suggestion
				if (file.endsWith('/index.html')) {
					// remove /index.html suffix
```

Reply:
I'm +1 for just commenting this rather than eliminating the magic number entirely.